### DISCUSSION TITLE
Inspiration for really fast marker clustering

### DISCUSSION THREAD

**Discussion by endlessuniverse** [fleaflet/flutter_map#1207] [Ideas]
https://github.com/SINTEF-9012/PruneCluster

**Comment by ibrierley** [comment]
There's certainly a lot of work gone into it, anyone feel free to convert ;).

Out of interest, there is a cluster package already, but I've not used it. Are there any drawbacks to the current one, i.e does it slow down on a lot of markers or something ?
